src/flask_restalchemy/resources/querybuilder.py

```diff
@@ -108,8 +108,11 @@ def parse_value(value, serializer):
     if not serializer:
         return value
     if isinstance(value, list):
-        return [serializer.load(item) for item in value]
-    return serializer.load(value)
+        return [
```

Comment:
Add a `session = serializer.model_class.query.session` in the beginning of the function, so you can get back to a one-liner list comprehension